src/formats/farbfeld.zig

```diff
@@ -0,0 +1,109 @@
+const std = @import("std");
+const builtin = @import("builtin");
+const mem = std.mem;
+const Allocator = std.mem.Allocator;
+const assert = std.debug.assert;
+const ImageUnmanaged = @import("../ImageUnmanaged.zig");
+const FormatInterface = @import("../FormatInterface.zig");
+const color = @import("../color.zig");
+const utils = @import("../utils.zig");
+const buffered_stream_source = @import("../buffered_stream_source.zig");
+
+pub const ReadError = ImageUnmanaged.ReadError;
+pub const WriteError = ImageUnmanaged.WriteError || error{BigDimensions};
+
+pub const Header = extern struct {
+    const width_size = 4;
+    const height_size = 4;
+    const size = 16;
+    const magic_value: [8]u8 = [8]u8{ 'f', 'a', 'r', 'b', 'f', 'e', 'l', 'd' };
```

Comment:
You can use a string directly here, it is not like C where a NULL terminator is automatically added at the end.